packages/discovery-react-components/package.json

```diff
@@ -11,6 +11,7 @@
     "test": "cross-env CI=1 react-scripts test",
     "test:watch": "react-scripts test --watch",
     "test:debug": "react-scripts --inspect-brk test --runInBand --no-cache",
+    "circular-dependencies": "madge --circular src/*",
```

Comment:
can we add this as part of our CI build? maybe just add it as part of the `lint` task

Reply:
Yeah, I mentioned in the PR description that I was considering putting in our build, although wasn't sure if we'd want it every time and increase the time the build takes. I think putting it in the `lint` task could be a good way to make sure it's run and checked with doing it every build though, I like that idea 👍